setup.py

```diff
@@ -42,7 +42,7 @@
     install_requires=[
         "pandas>=1.0,<2.0",
         "openpyxl>=3.0.6",
-        "snakemake>=7.22.0",
+        "snakemake>=7.32.4",
```

Comment:
Snakemake version 8.0 was released on December 20, so you were on the right track there. What's needed is not an updated *minimum* version, but an updated *maximum* version. You probably want something like `snakemake>=7.22,<8.0`.

Reply:
yup! that worked!